docs/source/configuration.rst

```diff
@@ -88,6 +109,79 @@ in projects which utilise a lot of complicated templating.
 
 .. _templateconfig:
 
+Rule Configuration
+------------------
+
+Rules can be configured with the :code:`.sqlfluff` config files.
+
+Common rule configurations can be set in the :code:`[sqlfluff:rules]` section.
+
+For example:
+
+.. code-block:: cfg
+
+   [sqlfluff:rules]
+   tab_space_size = 4
+   max_line_length = 80
+   indent_unit = space
+   comma_style = trailing
+   allow_scalar = True
+   single_table_references = consistent
+   unquoted_identifiers_policy = all
+
+Rule specific configurations are set in rule specific subsections.
+
+For example, enforce that keywords are upper case by configuring the rule
+:class:`L010 <sqlfluff.core.rules.Rule_L010>`:
+
+.. code-block:: cfg
+
+    [sqlfluff:rules:L010]  # Keywords
+    capitalisation_policy = upper
+
+All possible options for rule sections are documented in :ref:`ruleref`.
+
+For an overview of the most common rule configurations that you may want to
+tweak, see `Default Configuration`_ (and use :ref:`ruleref` to find the
+available alternatives).
+
+Enabling and Disabling Rules
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To disable individual rules, set :code:`exclude_rules` in the top level section
+of sqlfluff configuration. The value is a comma separated list of rule ids.
+
+For example, to disable the rules :class:`L010 <sqlfluff.core.rules.Rule_L022>`
+and :class:`L010 <sqlfluff.core.rules.Rule_L027>`:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    exclude_rules = L022, L027
+
+To enable individual rules, configure :code:`rules`, respectively.
+
+For example, to enable :class:`L010 <sqlfluff.core.rules.Rule_L027>`:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    rules = L027
+
+^ **TODO** does this make sense only if overriding an exclusion from a higher level
+configuration? Or what does this conf mean? How is it interpreted if
+:code:`exclude_rules = None` but :code:`exclude_rules != None`?
+
+Additionally, some rules have a special :code:`force_enable` configuration
+option, which allows to enable the given rule even for dialects where it is
+disabled by default. The rules that support this can be found in the
+:ref:`ruleref`.
+
+The default values can be seen in `Default Configuration`_.
+
+See also: :ref:`inline_ignoring_errors` and `Ignoring Errors & Files`_.
+(**TODO** aren't those two pretty much overlapping?)
+
```

Comment:
Should we also cross link back to the CLI args section from here?

Reply:
I don't think you need to cross link here. We're not cross linking to the configuration file for `rules` and `exclude_rules` so why cross link to CLI which has the same?

Reply:
I was thinking, this has now become the main place to read about "how to enable or disable rules". For completeness, it could be worth pointing to the fact that rules can be also controlled via CLI args. That could be the best option in some situation. But of course it is almost obvious that user would check what `sqlfluff lint --help` has to offer. I don't have any real opinion on what should be done. Adding links everywhere can of lead into a bit bloated docs, yes. So we live without linking to CLI args from here?

Reply:
Yeah I think leave it off here (for the bloated doc reason you give). It's not really the recommended way to ignore errors or include/exclude rules IMHO, but more a short hand for quite testing of only specific rules for that particular run.